velox/exec/tests/utils/ArbitratorTestUtil.h

```diff
@@ -86,6 +86,27 @@ struct TestAllocation {
   }
 };
 
+/// Scoped abort percentage utility that allows user to set the behavior of
```

Comment:
```
allows user to trigger abort during the query execution.
```
Shall we move this to QueryAssertions.h?

Reply:
OK, will fix it.